build.gradle

```diff
@@ -76,7 +76,7 @@ buildscript {
     }
 
     dependencies {
-        classpath "org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}"
+        classpath "org.springframework.boot:spring-boot-gradle-plugin:1.2.5.RELEASE"
```

Comment:
@adyach it's possible that we're doing something wrong, or that some change or another is documented in any version of spring-boot-gradle-plugin between 1.2.5 and 1.5.3. So, I first want to look at that. But I think this should be in another ticket.